src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -149,6 +150,17 @@ export default class DetailsBody extends React.Component<Props> {
 
     return (
       <StyledPageContent>
+        {incident &&
+          incident.status === IncidentStatus.CLOSED &&
+          incident.statusMethod === IncidentStatusMethod.RULE_UPDATED && (
+            <AlertWrapper>
+              <Alert type="warning" icon="icon-warning-sm">
```

Comment:
Could we use `IconWarning` here instead of the inlinesvg based icon?

`<Alert type="warning" icon={<IconWarning size="sm" />}>`

Reply:
Oh, cool. Didn't know we did this. Changed :-)